src/audiomixerboard.cpp

```diff
@@ -1423,7 +1437,17 @@ void CAudioMixerBoard::AutoAdjustAllFaderLevels()
 
             int group = vecpChanFader[i]->GetGroupID();
             if ( group == INVALID_INDEX )
-                group = 4;
+            {
+                if ( cntActiveGroups > 0 )
+                {
+                    // do not adjust the channels without group in group mode
+                    continue;
+                }
+                else
+                {
+                    group = 4;
```

Comment:
Again, should be `MAX_NUM_FADER_GROUPS`?

Reply:
This has also been fixed with commit 66aa3fc...